extra/resources/ClusterMon

```diff
@@ -153,7 +153,8 @@ ClusterMon_monitor() {
     if [ -f $OCF_RESKEY_pidfile ]; then
 	pid=`cat $OCF_RESKEY_pidfile`
 	if [ ! -z $pid ]; then
-	    kill -s 0 $pid >/dev/null 2>&1; rc=$?
+	    ps -wwfp $pid | grep $OCF_RESKEY_pidfile >/dev/null 2>&1 && \
+	        kill -s 0 $pid >/dev/null 2>&1; rc=$?
```

Comment:
Ok, the second question is in part answered by the existence of that very file.
But still, why not to start with checking `basename(argv[0])` of the process with the same PID, first?